src/py/flwr/server/driver/grpc_driver.py

```diff
@@ -157,39 +159,50 @@ class GrpcDriver(Driver):
         The version of the FAB used in the run.
     """
 
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
         self,
-        driver_service_address: str = DEFAULT_SERVER_ADDRESS_DRIVER,
-        root_certificates: Optional[bytes] = None,
-        fab_id: Optional[str] = None,
-        fab_version: Optional[str] = None,
+        run_id: int,
+        stub: Optional[GrpcDriverStub] = None,
```

Comment:
Looks good! The docstring above needs to be updated to reflect this

Reply:
Thanks! I've updated the doc string

Reply:
`driver_service_address` and `certificates` need to be removed as well